templates/aws/fips.html

```diff
@@ -10,474 +10,513 @@
-               class="p-button--positive"><span>Launch Ubuntu Pro FIPS 18.04 LTS on AWS</span></a>
-            <a href="https://aws.amazon.com/marketplace/pp/B08GCQS6D8?qid=1607612084948&sr=0-1&ref_=srh_res_product_title"
-               class="p-button--positive"><span>Launch Ubuntu Pro FIPS 16.04 LTS on AWS</span></a>
-          </p>
-        </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Ubuntu Pro FIPS for AWS compliance</h1>
+        <p class="p-heading--5">For AWS compliance in production environments.</p>
+        <p>
+          Ubuntu Pro FIPS is the first and only FIPS 140-2 certified image for AWS. Built upon the enhanced stability and security features of <a href="/aws/pro">Ubuntu Pro</a>, Ubuntu Pro FIPS is a critical foundation for federal programs and government contractors.
+        </p>
+        <hr class="is-muted" />
+        <ul class="p-inline-list u-responsive-realign">
+          <li class="p-inline-list__item">
+            <a class="p-button--positive"
+               href="https://aws.amazon.com/marketplace/pp/prodview-l2hkkatnodedk">Launch Ubuntu Pro FIPS 20.04 LTS on AWS</a>
+          </li>
+          <li class="p-inline-list__item">
+            <p>
+              <a href="https://aws.amazon.com/marketplace/pp/B08GCSCZJK">18.04&nbsp;&rsaquo;</a>
+            </p>
+          </li>
+          <li class="p-inline-list__item">
+            <p>
+              <a href="https://aws.amazon.com/marketplace/pp/B08GCQS6D8?qid=1607612084948&sr=0-1&ref_=srh_res_product_title">16.06&nbsp;&rsaquo;</a>
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div class="col">
+        <div class="p-image-container--16-9 is-highlighted">
           {{ image(url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
                     alt="",
-                    width="200",
-                    height="119",
+                    width="123",
+                    height="74",
                     hi_def=True,
-                    loading="auto",) | safe
+                    loading="auto",
+                    attrs={"class": "p-image-container__image"}) | safe
           }}
         </div>
+
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <h2>What's in Ubuntu Pro FIPS?</h2>
+    </div>
     <div class="row">
-      <div class="col-12">
-        <h2>What’s in Ubuntu Pro FIPS?</h2>
-        <ul class="p-matrix u-clearfix">
-          <li class="p-matrix__item">
-            <div class="p-matrix__content">
-              <h3 class="p-matrix__title">FIPS compliant by default</h3>
-              <div class="p-matrix__desc">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="is-muted" />
```

Comment:
Aren't we using p-rule--muted is such cases?

Reply:
We are indeed, thanks for flagging!